.tool-versions

```diff
@@ -0,0 +1,2 @@
+golang 1.21.8
```

Comment:
Can remove if needed: I'm using `asdf` to manager my nodejs/go versions.

Reply:
One concern is that this "duplicates" the source of truth that is now go.mod, at least as of Go v1.21. I'm only 0/5, though if this is useful!